README.md

```
@@ -221,171 +146,38 @@ fun BottomSheetPickerExample() {
+| Parameter | Description | Default |
+| :--- | :--- | :--- |
+| `state` | The state object to control the picker. | `rememberYearMonthPickerState()` |
+| `startLocalDate` | The initial date to set the picker to. | `currentDate` |
+| `yearItems` | List of years available for selection. | `1900..2100` |
+| `monthItems` | List of months available for selection. | `1..12` |
+| `visibleItemsCount` | Number of items visible in the list. | `3` |
 
-## 📄 License
+## License
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)